command-line/Lesson-3/README.md

```diff
@@ -86,7 +86,7 @@ OK, here are the git commands for the instructions above.  If you need help with
 
 1. From `main`, run `git checkout -b new-branch`
 2. Add the new file by right-clicking on your directory and selecting `New file` from the dropdown menu
-5. `git add goals.js` and `git commit -m "added my goals"`
+5. `git add Users` and `git commit -m "added my goals"`
```

Comment:
```suggestion
5. `git add Users/goals.js` and `git commit -m "added my goals"`
```

Reply:
I think it has to be `git add Users/DeliaConstantino/goals.js` (sub their name in the middle) if we want them to grab the specific file. Or we could use `git status`:
5. run `git status` and copy the file that shows up in red text (this is where changes were made)
6. `git add <copied file here>`
then back to the existing instructions with `git commit -m "commit message"`

I made some changes to try it out:
<img width="581" alt="Screen Shot 2021-11-09 at 4 33 48 PM" src="https://user-images.githubusercontent.com/70108137/141008501-287feeb2-fd1a-4066-8421-2c0af5142337.png">